components/feature/addons/src/main/res/layout/mozac_feature_addons_section_item.xml

```diff
@@ -3,15 +3,28 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <View
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:visibility="gone"
+        android:background="@android:color/darker_gray" />
```

Comment:
Should we use `android:background="?android:attr/colorListDivider"` ? This way we will support dark/light theme by default

Reply:
It is a good idea, although I think the list divider has a similar color for both themes. Your suggestion is better as we can support future changes in these themes.

Reply:
Seems like `colorListDivider ` is private, I used `listDivider` in change.